liberica/src/components/InputElements.tsx

```diff
@@ -1,3 +1,5 @@
+import React from "react";
```

Comment:
What's this for?

Reply:
This is required by the [react/react-in-jsx-scope](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md)
I think technically this is not required anymore since React 17, i will look into that later